_posts/andy_yang/2016-11-18-azure-java-examples.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: post
+title:  "Azure Blob Storage - Java Examples"
+breadcrumb: true
+author: andy_yang
+date: 2016-11-18
+categories: andy_yang
+tags:
+    - Azure
+    - Java
+    - Cloud
+    - Microsoft
+teaser:
+    info: Examples in Java for working with Azure Blob Storage
+    image: andy_yang/azure/azure-logo.jpg
+header: 
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+## Introduction
+The ICGC In The Cloud initiative makes a large genomic database available to cancer researchers in cloud computing environments. Cloud computing helps alleviate the burden, in transfer time and computing resources, that the size of the ICGC datasets places on research groups. Currently, over 750 TB of genomic data is hosted between Amazon Web Services and an OpenStack environment called Cancer Collaboratory. Recently, Microsoft has also expressed a willingness to join ICGC as a partner in hosting genomic data. 
```

Comment:
Cloud computing helps alleviate the burden, in transfer time and computing resources, that the size of the ICGC datasets places on research groups.
Cloud computing helps alleviate the burden, in transfer time and computing resources, on research groups caused by the size of the ICGC datasets.